src/content/docs/en/pages/guides/edge-application/ea-custom-ports/custom-ports.mdx

```diff
@@ -0,0 +1,111 @@
+
+1. In the **Origins** tab, select the default origin or add a new origin.
+2. Fill in the required fields.
+3. In the section **Origin Protocol Policy**, select **Enforce HTTP** if you want the edge to access your content explicitly through HTTP.
+4. Click the **Save** button.
+5. In the **Main Settings** tab of your edge application, select one of the default HTTP and HTTPS ports.
+6. Click the **Save** button.
+
+### Custom origin port to default application port
+
+You can use the `host:port` notation to specify the port from which your content will be retrieved and deliver it through a default application port. In this case, your content will be retrieved from a custom port of your origin and delivered through the default port you selected. To do so:
+
+1. In the **Origins** tab, select the default origin or add a new origin.
+2. Fill in the required fields.
+3. Under **Address**, type the hostname of your origin and add the port notation with the custom port. For example: `yourorigin.com:8080`.
+4. Keep the option Preserve HTTP/HTTPS protocol enabled.
+5. Click the **Save** button.
+6. In the **Main Settings** tab of your edge application, select one of the default HTTP or HTTPS ports.
+7. Click the **Save** button.
+
+### Default origin port to custom application port
+
+To deliver your content from a default origin port to a custom application port:
+
+1. In the **Origins** tab, select the default origin or add a new origin.
+2. Fill in the required fields.
+3. Under address, type the hostname of your origin. For example: `yourorigin.com`.
+4. Keep the option **Preserve HTTP/HTTPS protocol** enabled.
+5. Click the **Save** button.
+6. In the Main Settings tab of your edge application, select one of the custom HTTP or HTTPS ports. For example: `9443`.
+7. Click the **Save** button.
+
+### Custom origin port to custom application port
+
+If you want requests to be directed to an origin only using custom ports, you must specify that requests to your application made from the custom port are directed to the correct port. To do so:
+
+1. On the **Main Settings** tab, under **Ports**, check which custom ports have been selected.
+2. Go to the **Origins** tab.
+3. Click the **Add Origin** button.
+4. Give your origin an easy-to-remember name. For example: `Origin:8080`.
+5. Fill in the required fields.
+6. Under **Address**, use the `host:port` notation. For instance: **yourorigin.com:8080**.
+7. Keep the option **Preserve HTTP/HTTPS protocol** enabled.
+8. Fill in the remaining fields with the desired values.
+9. Click the **Save** button.
+
+Now you need to configure your application so that when requests are received in custom ports, they are redirected to the correct origin port:
```

Comment:
```suggestion
Now you need to configure your application so that when requests are received in custom ports, they're redirected to the correct origin port:
```